src/components/PageLevelBanner/PageLeveBanner.module.css

```
@@ -0,0 +1,89 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # PAGE LEVEL BANNER
+\*------------------------------------*/
+
+/**
+ * 1) Message of information, success, caution, or warning to the user
+ * 2) Position is relative to allow for absolute-positioned close button
+ * 3) Grid is used to separate the icon from the text with correct spacing
+ */
+.banner {
+  position: relative; /* 2 */
+  display: grid; /* 3 */
+  gap: var(--eds-size-2); /* 3 */
+  grid-template-columns: min-content 1fr; /* 3 */
+  padding: var(--eds-size-2) var(--eds-size-4);
+  background-color: var(--eds-theme-color-neutral-subtle-background);
+  border-bottom: var(--eds-border-width-md) solid var(--messaging-border-color);
+}
+
+/**
+ * PageLeveBanner informational icon
```

nit:
```suggestion
 * PageLevelBanner informational icon
```

and for the other css comments as well

---

lol I named the directory correctly but everything else is wrong 🤦🏼